sqlglot/parser.py

```diff
@@ -850,7 +852,6 @@ def _parse_partitioned_by(self):
 
     def _parse_stored(self):
         self._match(TokenType.ALIAS)
-        self._match(TokenType.EQ)
```

Comment:
why was this removed? i assume it's so that you can do stored="parquet" or something